### DISCUSSION TITLE
Input Icon

### DISCUSSION THREAD

**Discussion by samijuke** [bladewindui/ui#324] [Q&A]
Hello, 
I have a problem with the icon on the input field, the text passes over it.
is there a solution for this?
![image](https://github.com/user-attachments/assets/640f0003-e56b-4df1-94da-ccea2a62848a)

Have a nice day!

Sami

**Comment by mkocansey** [comment]
@samijuke Which version of BladewindUI are you on? This was fixed about 2 releases ago.

**Reply by samijuke** [reply]
Sorry, there was a conflict in my app.blade.php with Alpine.js. Multiple instances of Alpine.js were active in the project.
